src/main/java/com/org/jenkins/custom/jenkins/distribution/service/services/PackagerDownloadService.java

```diff
@@ -19,10 +20,11 @@
     private final static Logger LOGGER = Logger.getLogger(PackagerDownloadService.class.getName());
     private static Util util = new Util();
 
-    public ResponseEntity<Resource> downloadWAR(String versionName) throws Exception {
+    public ResponseEntity<Resource> downloadWAR(String versionName, String configuration) throws Exception {
         File warFile = null;
         String artifactId = getArtifactId();
         try {
+            WarGenerator.generateWAR(versionName, configuration);
             warFile = new File("/tmp/output/target/" + artifactId + "-" + versionName + ".war");
```

Comment:
Why hard coding the path? Is it possible to return the path from `generateWAR(...)`, or to pass the temporary file path from here so it does not have to be hard coded?

Reply:
Yeah I modified it to return the file